scalajvm/app/controllers/Quotes.scala

```diff
@@ -21,6 +23,42 @@ object Quotes extends Controller {
     Ok(views.html.index(quotes, pageNumber, countPages, order, filter))
   }
 
+  def getRSSFeed() = ActionWithTx { request =>
+    import request.dbSession
+    import play.api.Play.current
+
+    val deploymentURL =
+      current.configuration.getString("application.deploymentUrl").get
```

Comment:
There is no need for this deployment URL. You may want to take a look at how we generate the quote management URL in the email notifications: https://github.com/codingteam/loglist/blob/909960adfde5b2dd4c6713407019bfca6953b585/scalajvm/app/helpers/Notifications.scala#L35